### PR TITLE
YONK-323: move projects app to seperate repo

### PR DESCRIPTION
**Pull request by msaqib52**
As part of edx solutions apps consolidation work moving projects app into a separate repository.

@ziafazal please review.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
+projects-edx-platform-extensions
+================================
+
+projects-edx-platform-extensions (``projects``) is a Django application responsible for managing the concept of Projects in the Open edX platform. Projects are an intersection of Courses, CourseContent, and Workgroups.
```
Comment (ziafazal) [inline README.rst]: @msaqib52 We should mention in the readme that this projects app had dependency on organizations app so organizations app should be installed already to install projects app.

**Comment by mattdrayer** [comment]
@ziafazal -- I see we are including the tests with these repos, which is awesome, however I think it would be good to tie these code projects up to Travis or CircleCi in order to run the tests separately from edx-solutions/edx-platform

**Comment by ziafazal** [comment]
@mattdrayer these tests can not be run independently of edx-platform. They are very much dependent on edx-platform and also on other solutions apps that's why we did not include Travis integration with these repos. Having said that long term goal is to have these apps a independent services(IDA) and once we have ported them as IDAs we would integrate their repos with either Travis or CircleCi.

**Comment by msaqib52** [comment]
@ziafazal updated readme to mention organizations app dependency.

**Comment by ziafazal** [comment]
👍

**Pull request merged**